aws/cloudformation-templates/amazonpay.yaml

```diff
@@ -20,10 +20,6 @@ Parameters:
     Type: String
     Default: none
 
-  AmazonPayStoreId:
-    Type: String
-    Default: none
-
```

Comment:
Should this param be deleted from the [EE templates](https://github.com/aws-samples/retail-demo-store/tree/master/aws/cloudformation-templates/event-engine) too?

Reply:
It's still used in web-ui-pipeline.yaml for an env variable ... so I would think we still need it ?

https://github.com/aws-samples/retail-demo-store/blob/9b3324e4354acbc6b1a4c0f573fe97eb1bdb4dae/aws/cloudformation-templates/web-ui-pipeline.yaml#L404